runtimes/peregrine/Cargo.toml

```diff
@@ -122,6 +123,7 @@ runtime-benchmarks = [
   "pallet-treasury/runtime-benchmarks",
   "pallet-vesting/runtime-benchmarks",
   "pallet-utility/runtime-benchmarks",
+  "pallet-proxy/runtime-benchmarks",
```

Comment:
Same here, let's try to keep it sorted to find out more easily if something is missing.

Reply:
done